src/configuration.development.json

```diff
@@ -0,0 +1,54 @@
+{
+  "teamIds": [],
+  "groups": [],
+  "environment": "local",
+  "policy": "ephemeral",
+  "endpoints": {
+    "api": "https://automation.atomist.com/registration",
+    "graphql": "https://automation.atomist.com/graphql/team"
+  },
+  "http": {
+    "enabled": true,
+    "host": "localhost",
+    "port": 2866,
+    "auth": {
+      "basic": {
+        "enabled": false
+      },
+      "bearer": {
+        "enabled": false
+      }
+    },
+    "customizers": []
+  },
+  "ws": {
+    "enabled": true,
+    "termination": {
+      "graceful": false,
+      "gracePeriod": 60000
+    },
+    "compress": false
+  },
+  "applicationEvents": {
+    "enabled": false
+  },
+  "cluster": {
+    "enabled": false
+  },
+  "logging": {
+    "level": "debug",
+    "file": {
+      "enabled": true,
+      "level": "debug"
+    },
+    "banner": true
+  },
+  "statsd": {
+    "enabled": false
+  },
+  "commands": null,
```

Comment:
`null` and `undefined` are not treated the same by lodash, so this may prevent command and events from sources like atomist.config.js being merged in properly. There may not be a test for this.

Reply:
Of course, the custom handling of array merging may do the right thing here.

Reply:
I think I checked the null/undefined thing but maybe not. I'll check again.